src/scripts/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+
+  document.querySelector('html').addEventListener('click', () => {
+    resolve(`First promise was resolved`);
+  });
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.querySelector('html').addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.querySelector('html').addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    resolve('Second promise was resolved');
+  });
+});
+
+const thirdPromise = new Promise((resolve) => {
+  let RightClicked = false;
+  let LeftClicked = false;
```

Comment:
Variables must be in lowercase
```suggestion
  let rightClicked = false;
  let leftClicked = false;
```